.gitignore

```diff
@@ -41,3 +41,6 @@ build-storybook.log
 
 # Build artifacts
 dist/
+
+# env file
+.env
```

Comment:
i think this exists at the root level .env right?

Reply:
Okay to remove, I think it was autogenerated by something

Reply:
Hmm, don't quite get that, possible to clarify? Thanks!

Reply:
Thought this already exists, but saw it was missing.